mantidproject/mantid
# 'Old' MARI definition set sp4 as a monitor

For the old Mari definition we are missing a definiton for one of the monitors.  
This adds it in, using the location extracted from the RAW file using LoadInstrumentFromRaw.

This will only affect files before July 2017 as newer ones use another definition, and it does not have this monitor after those changes.

**To test:**
1. Start Mantid workbench,
1. Load MAR11060 (in test data)
1. Right click show detectors, sp 4 shold be a monitor
   ![image](https://user-images.githubusercontent.com/1017173/85011380-55eca800-b159-11ea-8ac6-12067c5b9643.png)
1.  right click and plot a colour fill plot, this spectra should not mess up the colour range so the image should look like this.
![image](https://user-images.githubusercontent.com/1017173/85011507-83d1ec80-b159-11ea-88d8-67488b4238f1.png)


This fine to merge, there are no associated code changes

Fixes #28845


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
